assets/scss/_nav.scss

```diff
@@ -99,41 +95,4 @@
             }
         }
     }
-}
-
-.navbar-position-absolute {
-    position: absolute !important;
-}
-
-// Icons
-#main_navbar {
-    li i {
-    padding-right: 0.5em;
-
-        &:before {
-            display: inline-block;
-            text-align: center;
-            min-width: 1em;
-        }
-    }
-  .alert {
-    background-color: inherit;
-    padding:0;
-    color: $secondary;
-    border: 0;
-    font-weight: inherit;
-
-        &:before {
-            display: inline-block;
-            font-style: normal;
-            font-variant: normal;
-            text-rendering: auto;
-            -webkit-font-smoothing: antialiased;
-            font-family: "Font Awesome 5 Free";
-            font-weight: 900; 
-            content: "\f0d9";
-            padding-left: 0.5em;
-            padding-right: 0.5em;
-        }
-    }  
-    }
```

Comment:
Moved into `_styles_project.scss`.

Reply:
After this move, `_nav.scss` matches the Docsy version of the file, so we'll be removing it in a followup PR. (I didn't do it here to make it more obvious which part is being moved into `_styles_project.scss`.